### PR TITLE
chore: release v0.22.0

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,54 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.22.0] - 2024-02-06
+
+Another breaking release where a new `ResponsePayload` type was introduced in order
+to make it possible to determine whether a response has been processed.
+
+Unforgently the `IntoResponse trait` was modified to enable that
```
Comment (lexnv) [inline CHANGELOG.md:12]: nit: Unfortunately?

**Review by lexnv — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (niklasad1) [inline CHANGELOG.md:12]:
```suggestion
Unfortunately the `IntoResponse trait` was modified to enable that
```

Comment (niklasad1) [reply CHANGELOG.md:12]: yes :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,54 @@ The format is based on [Keep a Changelog].
...
 
+## [v0.22.0] - 2024-02-06
+
+Another breaking release where a new `ResponsePayload` type was introduced in order
+to make it possible to determine whether a response has been processed.
+
+Unfortunately the `IntoResponse trait` was modified to enable that
+and some minor changes were made as well.
+
+### Example of async `ResponsePayload API`
```
Comment (niklasad1) [inline CHANGELOG.md:15]:
```suggestion
### Example of the async `ResponsePayload API`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,54 @@ The format is based on [Keep a Changelog].
...
+
+		rp
+	}
+}
+```
+
+### Roadmap
+
+We are getting closer to releasing jsonrpsee v1.0 and
+the following is work planned:
```
Comment (niklasad1) [inline CHANGELOG.md:42]:
```suggestion
the following work is planned:
```

**Review by jsdw — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,58 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.22.0] - 2024-02-06
```
Comment (niklasad1) [inline CHANGELOG.md:7]:
```suggestion
## [v0.22.0] - 2024-02-07
```

**Review by jsdw — APPROVED** [review]
Woop!

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,58 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.22.0] - 2024-02-07
+
+Another breaking release where a new `ResponsePayload` type was introduced in order
```
Comment (niklasad1) [inline CHANGELOG.md:9]:
```suggestion
Another breaking release where a new `ResponsePayload` type is introduced in order
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,58 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.22.0] - 2024-02-07
+
+Another breaking release where a new `ResponsePayload` type is introduced in order
+to make it possible to determine whether a response has been processed.
+
+Unfortunately the `IntoResponse trait` was modified to enable that
```
Comment (niklasad1) [inline CHANGELOG.md:12]:
```suggestion
Unfortunately, the `IntoResponse trait` was modified to enable that
```

**Pull request merged**